### PR TITLE
[docs] Add pricing links

### PR DESCRIPTION
**Pull request by oliviertassinari**
The pricing page is still a WIP with the design agency, for now, we can link the fulfillment page.

### CODE REVIEW

File: packages/x-license/src/licenseErrorMessageUtils.ts [hunk 1]
```diff
@@ -16,29 +16,29 @@ export function showInvalidLicenseError() {
...
-    'This is an evaluation only version, it is not licensed for',
-    'development projects intended for production.',
+    'This is an trial only version of Material-UI X.',
+    'While all the features are unlocked, it is not licensed for',
+    'development on projects intended for production.',
     '',
-    'If you want a license to use the features, please email',
-    'license@material-ui.com  for a trial license.',
+    'If you want a license to use the features, please visit',
+    'https://material-ui.com/r/x-license to get a valid license.',
```
Comment (oliviertassinari) [inline packages/x-license/src/licenseErrorMessageUtils.ts:33]: Require https://github.com/mui-org/material-ui/pull/22575

File: docs/src/pages/components/data-grid/getting-started/getting-started.md [hunk 2]
```diff
@@ -120,6 +121,11 @@ Please take the component for a test run, no need to contact us.
 If you have an enterprise grid running with an invalid license (no license, expired license, incorrect license) the grid displays a watermark and a warning in the console.
```
Comment (mbrookes) [inline docs/src/pages/components/data-grid/getting-started/getting-started.md:121]:
```suggestion
If you have an enterprise grid running with an expired or missing license key the grid displays a watermark, and a warning in shown in the console.
```

File: packages/x-license/src/licenseErrorMessageUtils.ts [hunk 3]
```diff
@@ -16,29 +16,29 @@ export function showInvalidLicenseError() {
...
 }
 
 export function showNotFoundLicenseError() {
   showError([
     'Material-UI X: License key not found.',
     '',
-    'All Material-UI X features are unlocked.',
-    'This is an evaluation only version, it is not licensed for',
-    'development projects intended for production.',
+    'This is an trial only version of Material-UI X.',
```
Comment (mbrookes) [inline packages/x-license/src/licenseErrorMessageUtils.ts:28]:
```suggestion
    'This is a trial-only version of Material-UI X.',
```

File: packages/x-license/src/licenseErrorMessageUtils.ts [hunk 4]
```diff
@@ -16,29 +16,29 @@ export function showInvalidLicenseError() {
...
+    'While all the features are unlocked, it is not licensed for',
+    'development on projects intended for production.',
```
Comment (mbrookes) [inline packages/x-license/src/licenseErrorMessageUtils.ts:30]:
```suggestion
    'development use on projects intended for production.',
```

File: packages/x-license/src/licenseErrorMessageUtils.ts [hunk 5]
```diff
@@ -16,29 +16,29 @@ export function showInvalidLicenseError() {
...
     '',
-    'If you want a license to use the features, please email',
-    'license@material-ui.com  for a trial license.',
+    'If you want a license to use the features, please visit',
```
Comment (mbrookes) [inline packages/x-license/src/licenseErrorMessageUtils.ts:32]:
```suggestion
    'To purchase a license, please visit',
```

**Review by mbrookes — CHANGES_REQUESTED** [review]

File: docs/src/pages/components/data-grid/getting-started/getting-started.md [hunk 6]
```diff
@@ -118,7 +118,7 @@ Please take the component for a test run, no need to contact us.
 
 ### Invalid license
 
-If you have an enterprise grid running with an invalid license (no license, expired license, incorrect license) the grid displays a watermark and a warning in the console.
+If you have an enterprise grid running with an expired or missing license key the grid displays a watermark, and a warning in shown in the console.
```
Comment (mbrookes) [inline docs/src/pages/components/data-grid/getting-started/getting-started.md:121]:
```suggestion
If you have an enterprise grid running with an expired or missing license key the grid displays a watermark, and a warning is shown in the console.
```

(Sorry)

**Pull request merged**